examples/generate_trials.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python
+
+import os, sys
+
+import logging
+import numpy as np
+import skyllh
+
+from skyllh.core.config import CFG
+from skyllh.core.random import RandomStateService
+from skyllh.core.timing import TimeLord
+from skyllh.core.minimizer import ScipyMinimizerImpl, LBFGSMinimizerImpl
+from skyllh.physics.source import PointLikeSource
+
+from i3skyllh.datasets import data_samples
+from i3skyllh.datasets import repository
+from i3skyllh.analyses.kdepdf_mcbg_ps.analysis import create_analysis
+
+from skyllh.core.analysis_utils import create_trial_data_file
+
+from skyllh.core.debugging import setup_logger, setup_console_handler
+setup_logger('skyllh', logging.INFO)
+setup_console_handler('skyllh', logging.INFO)
+
+import argparse
+parser=argparse.ArgumentParser()
+
+parser.add_argument("--config", "-c", help="Path to user config", type=str,
+                    required=True)
+parser.add_argument("--gamma", "-g", help="spectral index", type=float, default=2.0)
+parser.add_argument("--n_trials", "-nt", help="number of trials per job", type=int, default=10)
+parser.add_argument("--mean_ns", "-ns", help="mean number of nsig injected per job", type=float, default=0.0)
+parser.add_argument("--ncpus", "-nc", help="number of CPUs to use", type=int, default=1)
+parser.add_argument("--rss_seed", "-rs", help="random_number_seed", type=int, default=5)
+parser.add_argument("--gseed", "-gs", help="gamma seed for minimizer", type=float, default=3.0)
+
+args=parser.parse_args()
+print(args)
+
+# DO NOT USE PHOTOSPLINE.
+skyllh.core.pdf.PHOTOSPLINE_LOADED = False
+
+CFG.from_yaml(args.config)
+
+# Define datasets (todo: generalize to other analyses).
+bp = CFG["paths"]["base_path"]
+dsc_name = CFG["analysis"]["name"]
```

Comment:
I'm not sure if "analysis"/"name" is the right address for the definition of the dataset collection. It is indeed analysis specific, so I suggest "analysis"/"dsc_name"